R/read_sourmash_outputs.R

```diff
@@ -188,6 +215,10 @@ read_taxonomy_annotate <- function(file, intersect_bp_threshold = 0, separate_li
                                              lineage = readr::col_character())
 
   if(length(file) > 1){
+    # check if file or URL are valid, and if not, print helpful messages and errors.
```

Comment:
I wish there was a way to abstract these read methods so we didn't have to repeat all of this, but it's fine for now i guess

Reply:
I'll second that - even minimizing repetition would help to improve legibility at first pass, but probably not an essential change.